### DISCUSSION TITLE
Does pterm support serial port output?

### DISCUSSION THREAD

**Discussion by bingfengfeifei** [pterm/pterm#654] [Q&A]
I used pterm to write a menu tool that runs through a serial port connected device.
When I connect to the device via SSH or screen, the tool runs fine, but when connected via serial, it displays garbled characters.
Does pterm support running output through the serial port? Like the github.com/nsf/termbox-go project

**Comment by MarvinJWendt** [comment]
Hey, we do not guarantee support for serial port output. I would like to provide support, but sadly I don't have a way of reproducing this.

**Reply by bingfengfeifei** [reply]
I debug the serial port output by creating a virtual machine on vmware workstation and adding a virtual serial port. You can also use qemu to create a virtual machine and add virtual windows.
Then use a terminal connection tool to connect to reproduce the problem, such as using securityCrt or mobaxterm
